runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -211,6 +211,372 @@ doubleMaxMinHelper(TR::Node *node, TR::CodeGenerator *cg, bool isMaxOp)
+   uint8_t firstCallArgIdx = isStaticCall ? 0 : 1;
+
+   TR::Register* r_s1Value   = cg->evaluate          (node->getChild(firstCallArgIdx));
+   TR::Register* r_s1Len     = cg->gprClobberEvaluate(node->getChild(firstCallArgIdx+1));
+   TR::Register* r_s2Value   = cg->evaluate          (node->getChild(firstCallArgIdx+2));
+   TR::Register* r_s2Len     = cg->gprClobberEvaluate(node->getChild(firstCallArgIdx+3));
+   TR::Register* r_fromIndex = cg->gprClobberEvaluate(node->getChild(firstCallArgIdx+4));
+
+   // Registers
+   // r_tmp2 is used to eliminate data dependency in the VL-VL-VCEQ loop
+   TR::Register* r_resultReg       = cg->allocateRegister();
+   TR::Register* r_maxIndex        = cg->allocateRegister();
+   TR::Register* r_s1VecStartIndex = cg->allocateRegister();
+   TR::Register* r_s2VecStartIndex = cg->allocateRegister();
+   TR::Register* r_loadLen         = cg->allocateRegister();
+   TR::Register* r_tmp             = cg->allocateRegister();
+   TR::Register* r_tmp2            = cg->allocateRegister();
+   TR::Register* r_1stCharIndex    = cg->allocateRegister();
+
+   TR::Register* v_s1Partial       = cg->allocateRegister(TR_VRF);
+   TR::Register* v_s2Partial       = cg->allocateRegister(TR_VRF);
+   TR::Register* v_s2Char1Rep      = cg->allocateRegister(TR_VRF);
+   TR::Register* v_tmp             = cg->allocateRegister(TR_VRF);
+
+   // register dependencies
+   TR::RegisterDependencyConditions * regDeps = new (cg->trHeapMemory()) TR::RegisterDependencyConditions(0, 17, cg);
+   regDeps->addPostCondition(r_s1Value  , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_s1Len    , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_s2Value  , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_s2Len    , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_fromIndex, TR::RealRegister::AssignAny);
+
+   regDeps->addPostCondition(r_resultReg      , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_maxIndex       , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_s1VecStartIndex, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_s2VecStartIndex, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_loadLen        , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_tmp            , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_tmp2           , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(r_1stCharIndex   , TR::RealRegister::AssignAny);
+
+   regDeps->addPostCondition(v_s1Partial , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(v_s2Partial , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(v_s2Char1Rep, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(v_tmp       , TR::RealRegister::AssignAny);
+
+   // Labels
+   TR::LabelSymbol* label_find_1st_char        = TR::LabelSymbol::create(cg->trHeapMemory(), cg);
```

Comment:
Please use the `generateLabelSymbol` API for these to be consistent with the rest of the codebase.

Reply:
Fixed.